pkg/aws/dynamodb/dynamodb.go

```diff
@@ -2,24 +2,41 @@ package dynamodb
 
 import (
 	"context"
+	"strconv"
+	"strings"
 	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/cloudwatch"
 	"github.com/aws/aws-sdk-go/service/cloudwatch/cloudwatchiface"
 	"github.com/aws/aws-sdk-go/service/dynamodb"
 	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+	"github.com/aws/aws-sdk-go/service/pricing"
 
+	pricingWaste "github.com/cloudwaste/cloudwaste/pkg/aws/pricing"
 	util "github.com/cloudwaste/cloudwaste/pkg/aws/util"
 )
 
+type DynamoPricingFacet string
+
+const (
+	ReadCapacityUnitHours      DynamoPricingFacet = "ReadCapacityUnit-Hrs"
+	WriteCapacityUnitHours     DynamoPricingFacet = "WriteCapacityUnit-Hrs"
+	ReplWriteCapacityUnitHours DynamoPricingFacet = "ReplWriteCapacityUnit-Hrs"
+)
+
 const (
 	tableType = "DynamoDB Table"
 )
 
+var (
+	serviceCode = "AmazonDynamoDB"
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
`serviceCode` is unused (deadcode)